ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -132,13 +134,16 @@ export type AnalyticsEvent =
           };
       }
     /**
-     * Tracks each time the user applies a filter on a Workload page.
-     * This is controlled by the main search bar on all Workload CVE pages.
+     * Tracks each time the user applies a filter on a VM page.
+     * This is controlled by the main search bar on all VM CVE pages.
      * We only track the value of the applied filter when it does not represent
      * specifics of a customer environment.
      */
     | {
-          event: typeof WORKLOAD_CVE_FILTER_APPLIED;
+          event:
+              | typeof WORKLOAD_CVE_FILTER_APPLIED
+              | typeof NODE_CVE_FILTER_APPLIED
+              | typeof PLATFORM_CVE_FILTER_APPLIED;
```

Comment:
I was looking at this file and feel we should separate the constants and types into separate files for readability purposes. We don't need to do anything right now, but we should consider it as the file gets larger.

Reply:
Yes I agree. Maybe something like we are planning for the search filter configs? Each `Container` has its own set of exports that it defines?